Jenkinsfile

```diff
@@ -72,19 +74,24 @@ def RunIntegrationTest(k, v) {
         script{
             try{
                 withEnv(["KUBE_VERSION=${v}"]){
-                    unstash 'repo'
+                    unstash 'repo-amd64'
                     echo "running tests on k8s version ${v}"
+                    sh 'sleep 60' //intermittent issue when running apt-get right after vm is up
```

Comment:
i put a longer sleep just to be safe , only a few steps in kubeadm.sh script fails if its run immediately after slave mv is provisioned.  I will have to have the script smarter, but i wanted to defer it and not hold up 0.5